cuenca/resources/users.py

```diff
@@ -0,0 +1,87 @@
+from typing import ClassVar, List, Optional, cast
+
+from cuenca_validations.types import (
+    Address,
+    AddressUpdateRequest,
+    Beneficiary,
+    KYCFile,
+    KYCFileUpdateRequest,
+    PhoneNumber,
+    TOSAgreement,
+    TOSUpdateRequest,
+    UserQuery,
+    UserRequest,
+    UserStatus,
+    UserUpdateRequest,
+)
+from pydantic import EmailStr
+from pydantic.dataclasses import dataclass
+
+from ..http import Session, session as global_session
+from .base import Creatable, Queryable, Retrievable, Updateable
+from .identities import Identity
+from .resources import retrieve_uri
+
+
+@dataclass
+class User(Creatable, Retrievable, Updateable, Queryable):
+    _resource: ClassVar = 'users'
+    _query_params: ClassVar = UserQuery
+
+    identity_uri: str
+    platform_id: str
+    level: int
+    phone_number: PhoneNumber
+    email_address: EmailStr
+    profession: str
+    terms_of_service: Optional[TOSAgreement]
+    status: Optional[UserStatus]
+    address: Optional[Address]
+    govt_id: Optional[KYCFile]
```

Comment:
shouldn't this be a list of `KYCFile` to support front and back?

Reply:
El KYCFile tiene los atributos `front_uri` y `back_uri` por lo que no es necesario
https://github.com/cuenca-mx/cuenca-validations/blob/49f1eeb6df5ea717a3fd3cef5531a329c95137b7/cuenca_validations/types/identities.py#L47

Reply:
@matin   ^^